oo/classical-inheritance-es5.md

```diff
@@ -0,0 +1,164 @@
+Before the introduction of classses in ES6 it was common to mimic class inheritance with constructor Functions and [Object.create()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/create). Its also possible to use factory functions for this purpose. As these are not true classes we sometimes call these "pseudo-classes". The examples below show how to implement classical inheritance with the tools available in ES5:
```

Comment:
I wonder if it would be better to not mention ES6 as a comparison. Doing so mandates we cover one before the other. Would it be possible to isolate the discussion of comparisons to our lectures? It probably makes sense to expect they will know the ES5 way before the ES6 way, but not the other way around. Thoughts?